public/javascripts/controllers/admin-ctrl.js

```diff
@@ -1,8 +1,9 @@
 angular.module('jasmic.controllers')
-    .controller('AdministratorCtrl', ['$scope','$mdDialog', '$mdToast','UsersFactory','CropsFactory','SuppliersFactory',
+    .controller('AdministratorCtrl', ['$scope','$mdDialog', '$mdToast','$window','UsersFactory','CropsFactory','SuppliersFactory',
     'UserFactory','CropFactory','SupplierFactory', 'EmailFactory', 'UserProfileFactory', 'ParishesFactory',
```

Comment:
I haven't seen the behaviour described.  I also haven't seen the `$window.scroll` line u mentioned.  Could you point me to the file and line number on this branch?